assets/blocks/editor-components/confirm-dialog/confirm-dialog.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * WordPress dependencies
+ */
+import { Modal, Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useEffect } from '@wordpress/element';
+import { ENTER } from '@wordpress/keycodes';
+
+/**
+ * Controlled Component that shows a modal containing a confirm dialog. Inspired by Gutenberg's experimental
+ * Confirm Dialog.
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/components/confirm-dialog/
+ * @param {Object}   props           Component props.
+ * @param {boolean}  props.isOpen    Determines if the confirm dialog is open or not
```

Comment:
```suggestion
 * @param {boolean}  props.isOpen    Determines if the confirm dialog is open or not.
```

Just a missing period here.

Reply:
Good catch! Fixed in [0550301](https://github.com/Automattic/sensei/pull/5361/commits/0550301026edd985559e818d8b0e4fecd27f36b0).